src/App.tsx

```diff
@@ -1,172 +1,26 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { MainNavigation } from './components/MainNavigation';
+import { PeoplePage } from './pages/PeoplePage';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
 
 export const App = () => (
   <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
+    <MainNavigation />
 
     <main className="section">
       <div className="container">
```

Comment:
leave only `<Routes>`  tag in `App.tsx`

Create a `Layout` component and wrap all your routes in the `Layout` route.

```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside.

Reply:
why do I need that?

Reply:
code will looks better if we keep Route file(App.tsx), layout, and our pages separate.
 
 in case when i decide to add new page without `<MainNavigation/> ` what should i do? (i will just create new layout)